### PR TITLE
Add cros-ec test plan

### PR DESCRIPTION
**Pull request by ana**
Initial test plan that checks for /dev/cros_ec 

This allows to know if the driver probe has run successfully and the kernel has been able to establish communication with the embedded controller.

A test run: https://lava.collabora.co.uk/scheduler/job/1184530

### CODE REVIEW

File: templates/cros-ec/generic-uboot-tftp-ramdisk-cros-ec-template.jinja2 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% extends 'boot/generic-uboot-tftp-ramdisk-template.jinja2' %}
```
Comment (gctucker) [inline templates/cros-ec/generic-uboot-tftp-ramdisk-cros-ec-template.jinja2:1]: Maybe we could run this one on `peach-pi` (U-Boot)?

**Review by gctucker — COMMENTED** [review]
Worth checking on a U-Boot device, or remove the u-boot version if we only run this on Depthcharge.  All new Chromebook devices should be booting Depthcharge, so U-Boot is only for a few legacy ones.

File: templates/cros-ec/generic-uboot-tftp-ramdisk-cros-ec-template.jinja2 (hunk 1 above)
Comment (ana) [reply templates/cros-ec/generic-uboot-tftp-ramdisk-cros-ec-template.jinja2:1]: Done, test run at https://lava.collabora.co.uk/scheduler/job/1184534

**Comment by gctucker** [comment]
Thanks, testing this one on staging now.

**Pull request merged**